jerry-core/vm/vm.c

```diff
@@ -1046,8 +1046,14 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           continue;
         }
         case VM_OC_SET_PROPERTY:
+        case VM_OC_SET_UP_PROPERTY:
```

Comment:
Add a define around this, so this byte code is not executed when the feature is disabled.